Textures/meshes folder missing when installing using github

Thank you for creating the repo. It is extremely useful. I notice that installing using the github code (` pip install git+https://github.com/maximecb/gym-miniworld.git@ad817b83cfdf1dfac04e2d8871f58fa04214cd2f#egg=gym-miniworld`) does not include the `textures` and `meshes` directory. 

```
(mtrl) ➜  projects  ls -lh /private/home/sodhani/.conda/envs/mtrl/lib/python3.8/site-packages/gym_miniworld
total 85K
-rw-rw-r-- 1 sodhani sodhani  189 Jan  8 04:22 __init__.py
drwxrwsr-x 2 sodhani sodhani    0 Jan  8 04:22 __pycache__
-rw-rw-r-- 1 sodhani sodhani  13K Jan  8 04:22 entity.py
drwxrwsr-x 3 sodhani sodhani    0 Jan  8 04:22 envs
-rw-rw-r-- 1 sodhani sodhani 1.4K Jan  8 04:22 math.py
-rw-rw-r-- 1 sodhani sodhani  40K Jan  8 04:22 miniworld.py
-rw-rw-r-- 1 sodhani sodhani 8.7K Jan  8 04:22 objmesh.py
-rw-rw-r-- 1 sodhani sodhani  14K Jan  8 04:22 opengl.py
-rw-rw-r-- 1 sodhani sodhani 3.9K Jan  8 04:22 params.py
-rw-rw-r-- 1 sodhani sodhani 1.4K Jan  8 04:22 random.py
-rw-rw-r-- 1 sodhani sodhani 1005 Jan  8 04:22 utils.py
-rw-rw-r-- 1 sodhani sodhani 1.3K Jan  8 04:22 wrappers.py
```

This leads to errors like `AssertionError: failed to load textures for name "brick_wall"`

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,0 +1,2 @@
+include gym_miniworld/meshes/*
+include gym_miniworld/textures/*

